<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="10188" started-at="2021-11-03T09:32:57Z" finished-at="2021-11-03T09:33:07Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="10188" started-at="2021-11-03T09:32:57Z" finished-at="2021-11-03T09:33:07Z">
      <class name="TestCases.HomeTestCase">
        <test-method status="PASS" signature="HomePageAdminOperation()[pri:0, instance:TestCases.HomeTestCase@5e5792a0]" name="HomePageAdminOperation" duration-ms="2008" started-at="2021-11-03T15:03:02Z" finished-at="2021-11-03T15:03:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HomePageAdminOperation -->
      </class> <!-- TestCases.HomeTestCase -->
      <class name="TestCases.newUserTestCase">
        <test-method status="SKIP" signature="registerUser(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestCases.newUserTestCase@3532ec19]" name="registerUser" duration-ms="0" started-at="2021-11-03T15:03:07Z" data-provider="Data" finished-at="2021-11-03T15:03:07Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:163)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1274)
	at org.testng.internal.Invoker.createParameters(Invoker.java:989)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1079)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
	at ExcelOpeartions.ExcelOperation.readdata(ExcelOperation.java:60)
	at TestCases.newUserTestCase.data(newUserTestCase.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerUser -->
      </class> <!-- TestCases.newUserTestCase -->
      <class name="TestCases.LoginTestCase">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:TestCases.LoginTestCase@3ffc5af1]" name="openBrowser" is-config="true" duration-ms="7876" started-at="2021-11-03T15:02:49Z" finished-at="2021-11-03T15:02:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="Login()[pri:0, instance:TestCases.LoginTestCase@3ffc5af1]" name="Login" duration-ms="5039" started-at="2021-11-03T15:02:57Z" finished-at="2021-11-03T15:03:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
      </class> <!-- TestCases.LoginTestCase -->
      <class name="TestCases.systemUserTest">
        <test-method status="PASS" signature="Adduser()[pri:0, instance:TestCases.systemUserTest@26653222]" name="Adduser" duration-ms="956" started-at="2021-11-03T15:03:05Z" finished-at="2021-11-03T15:03:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Adduser -->
      </class> <!-- TestCases.systemUserTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
